src/List/ListItem.js

```diff
@@ -41,6 +42,7 @@ function getStyles(props, context, state) {
       !state.rightIconButtonKeyboardFocused ? hoverColor : null,
       color: textColor,
       display: 'block',
+      cursor: disabled ? 'not-allowed' : 'inherit',
```

Comment:
I don't think that we should apply a `not-allowed` cursor here.
As for the `DateDisplay.js`, I think that we should apply it when the style of the component change, to make it even more obvious that the interaction is disabled.
Here, we do not change the style, we disable the ripples. Hence, I would rather keep the default cursor.
Does that makes sens?